packages/bindings-test/src/multitest.rs

```diff
@@ -199,28 +225,12 @@ impl Module for OsmosisModule {
                     return Err(OsmosisError::Unimplemented.into());
                 }
                 let mut pool = POOLS.load(storage, first.pool_id)?;
+                let payout =
+                    pool.swap_with_limit(&first.denom_in, &first.denom_out, amount.clone())?;
                 let (pay_in, get_out) = match amount {
-                    SwapAmountWithLimit::ExactIn { input, min_output } => {
```

Comment:
Nice to pull this out into it's own function

Reply:
OK, I'll keep this and never the `WithLimit` thing.